arrows/serialize/json/load_save_metadata.cxx

```diff
@@ -67,32 +66,50 @@ struct meta_item
       {
         const bool value = kwiver::vital::any_cast< bool > ( this->item_value );
         archive( CEREAL_NVP( value ) );
+        type = "boolean";
+      }
+      // We don't want negative ints to be serialized as large postive unsigned
+      // values since this would be confusing for external applications
+      else if ( trait.is_signed() )
+      {
+        const int value = kwiver::vital::any_cast< int > ( this->item_value );
+        archive( CEREAL_NVP( value ) );
+        type = "integer";
       }
       else
       {
         const uint64_t value = kwiver::vital::any_cast< uint64_t > ( this->item_value );
         archive( CEREAL_NVP( value ) );
+        type = "unsigned integer";
       }
     }
     else if ( trait.tag_type() == typeid( std::string ) )
     {
       const std::string value = kwiver::vital::any_cast< std::string > ( this->item_value );
       archive( CEREAL_NVP( value ) );
+      type = "string";
     }
     else if ( trait.tag_type() == typeid( kwiver::vital::geo_point ) )
     {
       const kwiver::vital::geo_point value = kwiver::vital::any_cast<kwiver::vital::geo_point  > ( this->item_value );
       archive( CEREAL_NVP( value ) );
+      type = "geo-point";
     }
     else if ( trait.tag_type() == typeid( kwiver::vital::geo_polygon ) )
     {
       const kwiver::vital::geo_polygon value = kwiver::vital::any_cast<kwiver::vital::geo_polygon  > ( this->item_value );
```

Comment:
I know this is not really part of your change, but might be useful to break long lines here or use `auto` and also remove extra whitespace in `geo_polygon  >`

Reply:
:+1:, balancing the whitespace in `<>` would be nice also. (It looks like it is probably okay to follow the style guide's prescription to include space inside `<>`, as the surrounding code seems to generally use space inside open/close symbols.)